### PR TITLE
Always use a static back route

### PR DESCRIPTION
**Pull request by xoxys**
Fixes: https://github.com/woodpecker-ci/woodpecker/issues/1998

I don't know if there is a situation where using the browser history makes sense, or if we should always use a static route for the back buttons, but this fix at least preserves the ability to use the browser history.

### CODE REVIEW

**Comment by 6543** [comment]
no it makes no sense to use browser history

**Comment by 6543** [comment]
@xoxys can you refactor it to drop browser histry func

**Comment by xoxys** [comment]
Ok so I will rewrite the function to always use the given route.

File: web/src/views/org/OrgSettings.vue [hunk 1]
```diff
@@ -41,5 +41,5 @@ onMounted(async () => {
   }
 });
 
-const goBack = useRouteBackOrDefault({ name: 'repos-owner' });
+const goBack = useRouteBack({ name: 'repos-owner' });
```
Comment (6543) [inline web/src/views/org/OrgSettings.vue:44]: we should use the orgID route here :)

Comment (6543) [reply web/src/views/org/OrgSettings.vue:44]: yes that's also an bug :sweat:

File: web/src/views/repo/RepoSettings.vue [hunk 2]
```diff
@@ -73,5 +73,5 @@ onMounted(async () => {
   }
 });
 
-const goBack = useRouteBackOrDefault({ name: 'repo' });
+const goBack = useRouteBack({ name: 'repo' });
```
Comment (6543) [inline web/src/views/repo/RepoSettings.vue:76]: same here ... we should use the repoID route

File: web/src/views/repo/pipeline/PipelineWrapper.vue [hunk 3]
```diff
@@ -196,5 +196,5 @@ const activeTab = computed({
   },
 });
 
-const goBack = useRouteBackOrDefault({ name: 'repo' });
+const goBack = useRouteBack({ name: 'repo' });
```
Comment (6543) [inline web/src/views/repo/pipeline/PipelineWrapper.vue:199]: same ... repoID

**Review by 6543 — APPROVED** [review]

**Pull request merged**